e2e/utils.ts

```diff
@@ -174,3 +173,9 @@ export function releaseLock(lockID: string) {
     // eslint-disable-next-line no-console
     console.log(test.info().title, 'lock released');
 }
+
+export async function getHTTPHeaders(context: APIRequestContext) {
+    return {
+        'X-CSRF-Token': (await context.storageState()).cookies[2].value,
```

Comment:
is there any chance this (using an index) will break?

Reply:
@cpoile That's for you to figure out. I did my part :p

It shouldn't be hard to convert this into a more robust "search" function.

Reply:
equivalent to starting fires before turning the lights out. 😝

Reply:
@cpoile I could have left this ticket in the backlog, and in a couple of months, everything would have broken down due to this (v11)

Reply:
I was kidding, but I know, I appreciate it